### PR TITLE
Added gaming layout and read me

### PR DESCRIPTION
**Pull request by rafael-azevedo**
## Types of Changes

- [X] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  _BASE,
+  _GAME,
+  _FUNC,
+  _RGB
+};
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_BASE] = LAYOUT(
+  KC_ESC, KC_GRV,  KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_PGUP,  KC_PGDN, \
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c]:
```suggestion
  KC_ESC, KC_GRV,  KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_PGUP,  KC_PGDN,
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ KC_HOME, KC_TAB,  KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, \
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c]:
```suggestion
 KC_HOME, KC_TAB,  KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ KC_END,  CAPSTL,  KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L,    KC_SCLN, KC_QUOT,          KC_ENT, \
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c]:
```suggestion
 KC_END,  CAPSTL,  KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+          KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT,  KC_SLSH, KC_DEL,  MO(2), \
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c]:
```suggestion
          KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT,  KC_SLSH, KC_DEL,  MO(2),
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+          TG(1),      KC_LAPO, KC_BSPC, KC_LGUI,     KC_SPC,       KC_RAPC,                   MO(3) \
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/keymap.c]:
```suggestion
          TG(1),      KC_LAPO, KC_BSPC, KC_LGUI,     KC_SPC,       KC_RAPC,                   MO(3)
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/readme.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+## Gaming Layout
+
+### This layout was designed for everyday work with all necessary keys reachable from the home row, while having a toggle layer moving the space and backspace to a more traditional location. 
+
+* The main layer gives you access to home, end , pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row.
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/readme.md]:
```suggestion
* The main layer gives you access to home, end, pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row. 
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/readme.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+## Gaming Layout
+
+### This layout was designed for everyday work with all necessary keys reachable from the home row, while having a toggle layer moving the space and backspace to a more traditional location. 
+
+* The main layer gives you access to home, end , pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row. 
+* The main layer also includes capslock key while tapped and it becomes control when pressed. 
+* **Toggle(1)[Game]:** Toggles into a gaming layout that gives you space on your left hand thumb and moves the backspace to the traditional location. That way you can game one handed while using a mouse.  
+* **MO(2)[_FUNC]:** Is a momentary layer for all special functions including F keys , volume control, and print screen. It also keys character C and V, allowing you to copy and paste while using arrow keys. This is an easy to reach location for all special funtionaliy that is not normally used. 
+* **M(3)[_RGB]:** Is a monentary layer that gives you all the RGB control and keyboard Reset. This moves the functionality not normally used to a place you won't accidently hit and get annoyed at.
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/readme.md]:
```suggestion
* **M(3)[_RGB]:** Is a momentary layer that gives you all the RGB control and keyboard Reset. This moves the functionality not normally used to a place you won't accidentally hit and get annoyed at.
```

File: keyboards/maartenwut/wonderland/keymaps/gaming/readme.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+## Gaming Layout
+
+### This layout was designed for everyday work with all necessary keys reachable from the home row, while having a toggle layer moving the space and backspace to a more traditional location. 
+
+* The main layer gives you access to home, end , pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row. 
+* The main layer also includes capslock key while tapped and it becomes control when pressed. 
+* **Toggle(1)[Game]:** Toggles into a gaming layout that gives you space on your left hand thumb and moves the backspace to the traditional location. That way you can game one handed while using a mouse.  
+* **MO(2)[_FUNC]:** Is a momentary layer for all special functions including F keys , volume control, and print screen. It also keys character C and V, allowing you to copy and paste while using arrow keys. This is an easy to reach location for all special funtionaliy that is not normally used.
```
Comment (ridingqwerty) [inline keyboards/maartenwut/wonderland/keymaps/gaming/readme.md]:
```suggestion
* **MO(2)[_FUNC]:** Is a momentary layer for all special functions including F keys , volume control, and print screen. It also keys character C and V, allowing you to copy and paste while using arrow keys. This is an easy to reach location for all special funtionality that is not normally used. 
```

**Comment by rafael-azevedo** [comment]
Updated all the spelling and format issues. I left the enum in the layers because I want to ensure that order.

**Comment by drashna** [comment]
The only real issue I see here, is that usually we want to have the keymap folder name match your github username, as this makes tracking ownership a lot easier for us. 

This isn't a hard requirement, I think, but it is highly encouraged.

**Comment by rafael-azevedo** [comment]
@drashna  moved the keymap to a folder matching my github name so that it matches current standards.

**Pull request merged**